lib/HttpSignatureStrategy.js

```diff
@@ -11,7 +11,10 @@ var passport = require('passport');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
 var didio = require('did-io');
+var jsonld = require('jsonld');
 var URL = require('url');
+var request = require('request');
+request = request.defaults({json: true, strictSSL: false});
```

Comment:
Note that "json: true" isn't quite what we want here. We want to actually request `application/ld+json; application/json`. That means we have a preference for JSON-LD first, but we'll take JSON if that's not available.